packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.classNames.ts

```diff
@@ -1,28 +1,118 @@
 import { memoizeFunction } from '../../Utilities';
-import { mergeStyleSets, ITheme } from '../../Styling';
-import { IContextualMenuStyles } from './ContextualMenu.Props';
+import { mergeStyles } from '../../Styling';
+import { IContextualMenuStyles, IMenuItemStyle } from './ContextualMenu.Props';
 
 export interface IContextualMenuClassNames {
-  title?: string;
+  container: string;
+  root: string;
+  list: string;
+  divider: string;
+  header: string;
+  title: string;
+}
+
+export interface IMenuItemClassNames {
+  item: string;
+  root: string;
+  linkContent: string;
+  icon?: string;
+  subMenuIcon?: string;
+  label: string;
 }
 
 export const getClassNames = memoizeFunction((
   styles: IContextualMenuStyles,
-  theme: ITheme,
+  className: string
 ): IContextualMenuClassNames => {
 
   styles = styles || {};
 
-  return mergeStyleSets({
-    title: [{
-      fontSize: '16px',
-      paddingRight: '14px',
-      paddingLeft: '14px',
-      paddingBottom: '5px',
-      paddingTop: '5px',
-      backgroundColor: theme.palette.neutralLight
-    },
-    styles.title
-    ]
-  });
+  return {
+    container: mergeStyles(
+      'ms-ContextualMenu-container',
+      className,
+      styles.container
+    ) as string,
```

Comment:
You don't need the as string anymore after 5.0, please remove them here and in other places in this file.